docs/source/release.rst

```diff
@@ -50,115 +64,14 @@ file which is the single source of truth for version information. We try to
 follow SemVer, so typically a release will involve changing the version
 ``vX.Y.Z`` to ``vX.(Y+1).Z``, constituting a MINOR version increase.
 
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-Generate the HTML and PDF file for the docs
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-To create the HTML documentation, run the following from the top-level
-directory of the repository:
-
-.. code-block:: shell-session
-
-    make docs
-
-To create the PDF documentation, do the following:
-
-.. code-block:: shell-session
-
-    make pdf
-
-Finally, Since the ``docs/build`` folder is not version controlled, copy the
-newly created PDF file from ``docs/build/latex`` to ``docs/pdf`` folder as
-``mitiq.pdf`` to overwrite the previous version.
-
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-Create a PR with the above changes
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-After the required changes to ``VERSION.txt`` and ``CHANGELOG.md`` have been
-made, and the PDF documentation has been generated and moved to the correct
-location, it is recommended that the release author make a PR to master with
-these changes (rather than pushing directly to master) just in case. After
-this PR has been merged, the release author can go to the next step.
-
-^^^^^^^^^^^^^^^^
+----------------
 Create a new tag
-^^^^^^^^^^^^^^^^
+----------------
 
-Tag the new commit to master (using ``git tag``) with a tag that matches the
+Tag the new commit to release (using ``git tag``) with a tag that matches the
```

Comment:
```suggestion
Tag the new commit to ``release`` (using ``git tag``) with a tag that matches the
```

Reply:
I understand ``release`` will be the name of the branch for all of them. If not, maybe this could be clarified.

Reply:
Does the tagging automatically prompt the PR merging, or one needs to merge it? This could be specified.

Reply:
I have added more clarification about both of these.